src/id8_i/configs/aerotech_stages_devices.yml

```diff
@@ -0,0 +1,14 @@
+id8_i.devices.aerotech_stages.AerotechSampleStage:
+- name: sample
+  prefix: "8idiAerotech:"
+  labels: ["sample", "stage"]
+
+id8_i.devices.aerotech_stages.AerotechDetectorStage:
+- name: detector
+  prefix: "8idiAerotech:"
+  labels: ["detector", "stage"]
```

Comment:
One purpose of `labels` is to support the Bluesky Magics `%wa` and`%ct`.  When last I looked, the `detectors` (plural) was the label they were [advising](https://github.com/bluesky/bluesky/blob/f46b296dd74236f4402e61d4a33665df7f324300/src/bluesky/magics.py#L152-L155).  If this stage is not an actual detector (but only a stage that supports a detector), remove the `detector` label here or change it to avoid confusion.